main.rb

```diff
@@ -1,9 +1,7 @@
-require_relative 'person'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
+require_relative 'app'
 
-person = Person.new(22, name: 'maximilianus')
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+def main
+  app = App.new
+  app.run
+end
```

Comment:
- [ ] Hi @enocol, this method(`main`) is in a file called main.rb. A method in ruby bundles one or more statements. In this project, they have listed out and presented to us the statements(In the form of a list) this method(`main`) should bundle(contain/hold). The previous reviewer gave you hints on what should be in this method.
![Screenshot from 2023-11-18 11-34-07](https://github.com/enocol/old_school_library/assets/63087491/bf921251-b81e-4838-ab15-27378d4e9ab0)

- I think we should kindly considered a review as a way of learning to grow as professionals. Based on the hints provided by @topeogunleye, it should take you less than 5 mins to fix this issue if you're willing to learn

Reply:
Hi @FrederickMih A method in runy bundles one or more statements. Correct! Did they project requirement say that the should be a particular statement in the method? Has the method performed the tast or not should the question.  I am confued or may we are talking about something  i do not yet understand

Reply:
Hi,

I hope you're doing well. The feedback from the code reviewers suggests that it would be beneficial to add the code for starting the app, including the menu items, directly in the main.rb file. This would help meet the project requirements more effectively.

If you have reservations about this approach, it's completely understandable. However, I would recommend considering this feedback and, if needed, feel free to request a second opinion or further clarification. The goal is to ensure that the implementation aligns well with the specified requirements.

If you have any questions or concerns, don't hesitate to reach out.

Best regards,
Temitope

Reply:
@FrederickMih I have file called main.js. This file has a method that when invoked, starts up the application presenting the use with those options. OBJECTIVES TO BE FULFILLED IN THE MAIN ARE:
PRESENT THE USER WITH THE LIST OF OPTIONS to choose (not present the user with the code)

Reply:
- [ ] @enocol, list of options are generated from codes though :smile:

Reply:
@Nemwel-Boniface I need to discuss this requirement with you to fully understand because what you have said is just confirming without any proof. I  need to discus this with you so that both of us can make sense out of this. please lets meet on slack or zoom chat. I am waiting. I am online now. So that we can both evaluate what this requirements are based on what you have confirmed

Reply:
@Nemwel-Boniface . Its been 1 hour since tagged you as you mention to get things clarified. Yet you have not responded. I have send you a message on zoom, on slack yet no response. I need more information from you in order to make sure we are on the same page bacause, i have been looking a that project requirement several times to see where i must have missed something but yet could not. You feedback after your review is very scanty and lack enough evidence to support your decision. I have started wondering if we are all talking about the same project